src/Subscribers/AuthenticationLogSubscriber.php

```diff
@@ -41,6 +41,16 @@ public function handleAuthenticatableLogout($event)
         ]);
     }
 
+    public function handleAuthenticatableLockout($event)
+    {
+        AuthenticationLogRecord::create([
+            'credentials' => Omissions::omitCredentials($event->request->query()),
+            'eventType' => get_class($event),
+            'user_ip' =>  $event->request->getClientIp,
+            'recorded_at' => now()
```

Comment:
While this isn't essential and I don't normally penalise PRs because they're missing, we do prefer all array contents end in a comma:
```suggestion
            'recorded_at' => now(),
```
As I'm sure you know, this will make it much easier to move the array entries around (change their order) and is one less thing to think about when you need to add another item to the array down the line